docker-compose.yml

```diff
@@ -12,6 +12,8 @@ services:
   dj:
     container_name: dj
     stdin_open: true
+    env_file:
+      - ./datajunction-server/.docker-env/.env
```

Comment:
@shangyian this .env file was moved up one directory to `./datajunction-server/.env`. Also since it's in the root of the build context directory it should just get picked up automatically without this line.

Reply:
Oh good point! Let me remove this in a new PR